### PR TITLE
RED-37763 Revise hardware requirements to refer to each node

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

File: content/rs/administering/designing-production/hardware-requirements.md [hunk 1]
```diff
@@ -31,9 +31,9 @@ requirements for a production system:
 
 | Item | Description | Minimum Requirements | Recommended |
 |------------|-----------------|------------|-----------------|
-| # of nodes per cluster  | At least 3 nodes are needed in order to support a reliable, highly available deployment that handles process failure, node failure, and network split events in a consistent manner. | 3 nodes | >=3 nodes; must be odd number of nodes to maintain quorum |
+| # of nodes per cluster  | At least 3 nodes (2 data nodes and 1 quorum node) are needed in order to support a reliable, highly available deployment that handles process failure, node failure, and network split events in a consistent manner. | 3 nodes | >=3 nodes; must be odd number of nodes to maintain quorum |
```
Comment (Paz-Yanover) [inline content/rs/administering/designing-production/hardware-requirements.md]: We don't need to mention "quorum node" here, as it's not a requirement to have a quorum node, and the cluster can have 3 "regular" nodes

File: content/rs/administering/designing-production/hardware-requirements.md [hunk 2]
```diff
@@ -31,9 +31,9 @@ requirements for a production system:
...
+| RAM per node | Defining your RAM size should be part of the capacity planning for your Redis usage. Since Redis uses a relatively large amount of buffers (i.e. for slave communication, for the client communication and for pub/sub commands) the operator should take extra care to maintain at least 30% of the RAM "unused" on each node.</br>Additional recommendations:</br>1. If some of the cluster nodes are utilizing more than 65% of the RAM, you should look at migrating busy resources to less busy nodes.</br>2. If all the cluster nodes are utilizing over 70% of the RAM, you should look to scale out the cluster by [adding a node]({{< relref "/rs/administering/cluster-operations/adding-node.md" >}}).</br>3. Do not run other memory-consuming systems on the same machine that is used as an RS node. | 15GB | >=30GB |
```
Comment (Paz-Yanover) [inline content/rs/administering/designing-production/hardware-requirements.md]: @bmansheim - the change is only the addition of "per node"? 
Should we do the same also for "cores"

**Comment by bmansheim** [comment]
@ushachar Should we add some guideline for the amount of storage in a production environment?

**Comment by ushachar** [comment]
> @ushachar Should we add some guideline for the amount of storage in a production environment?

Less a question for me... more for product.

**Comment by bmansheim** [comment]
@Paz-Yanover @AlonMagrafta Should we add some guideline for the amount of storage in a production environment?

**Comment by bmansheim** [comment]
@AlonMagrafta @Paz-Yanover I removed the Storage row because it's just a parent for the other storage rows and not an actual requirement.

**Pull request merged**